src/hotspot/share/utilities/globalDefinitions.hpp

```diff
@@ -92,6 +92,12 @@ class oopDesc;
 
 #define BOOL_TO_STR(_b_) ((_b_) ? "true" : "false")
 
+// Format 8-bit quantities.
+#define PTR8_FORMAT            "0x%02" PRIx8
+
+// Format 16-bit quantities.
+#define PTR16_FORMAT           "0x%04" PRIx16
+
```

Comment:
nit: I would find UINT(8|16)_FORMAT_X more logical.

Reply:
See my comments to David's comments.